### ISSUE TITLE
Cmake failing for cgal 4.12 (brew) on mac

### ISSUE THREAD

**Issue by sgsellan** [libigl/libigl#794]
As of today, running `brew install cgal´on a Mac installs CGAL version 4.12. When attempting to use libigl for cmake with this version of CGAL, one gets error: 

"-- CGAL not found.  Set the CGAL_DIR cmake variable or environment variable to the directory containing CGALConfig.cmake. This is either the binary directory where CGAL was configured or PREFIX/lib/CGAL for an installation."

Thanks!

**Comment by jdumas** [comment]
Oh, apparently they modernized their CMake a bit more with the 4.12 release. Maybe there is hope after all :p 

I'll have to take a better look at this after the sigasia deadline (next week).

For now you can try to install locally an older version following [this](https://github.com/libigl/libigl/blob/master/external/CMakeLists.txt), or see if you can fix the current build.

**Comment by sgsellan** [comment]
Yeah, installing an older version seems to solve it, just wanted to let you guys know! Thanks for taking a look!

**Comment by jdumas** [comment]
Sure, thanks for reporting. Will fix it whenever possible =)

**Comment by alecjacobson** [comment]
For those that installed CGAL 4.12 via homebrew on Mac, a temporary fix is:

```
brew unlink cgal
% Force install 4.11 using old brew recipe
brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/3fae804bc40288bca3921f81d0429ea9b2425a8e/Formula/cgal.rb
```

**Comment by jdumas** [comment]
Just FYI that I have a PR #832 that updates the CGAL dependency to 4.12, for those who want to try out CGAL 4.12+ with libigl.
